qa/suites/fs/thrash/multifs/tasks/1-thrash/mgr.yaml

```diff
@@ -0,0 +1,10 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - overall HEALTH_
+      - \(MGR_DOWN\)
+      - cluster \[WRN\] evicting unresponsive client
+tasks:
+  - mon_thrash:
```

Comment:
```suggestion
  - mgr_thrash:
```

?

Reply:
What is the rationale for putting it in `fs:thrash:multifs` anyway?

Reply:
> What is the rationale for putting it in `fs:thrash:multifs` anyway?

wouldn't it help running the mgr thrasher with multifs workloads with this placement of mgr.yaml under the multifs dir ? or it wouldn't matter since its available under workloads dir ?

Reply:
I just don't see the mgr getting exercised in `fs:thrash:multifs` so I don't know what the point is.